editor/utils/map.ts

```diff
@@ -56,18 +56,35 @@ export function useResizeObserver<T extends HTMLElement>(
  * Returns undefined if the geojson has no features
  */
 export const useCurrentBounds = (
-  geojson: FeatureCollection
+  geojson: AllGeoJSON
```

Comment:
this is a cool TIL that turf has this type. idk how strict we are being about types in here but would it makes sense to limit the type to just Feature and FeatureCollection?

Reply:
updated to remove the manual bbox construction mentioned in this comment https://github.com/cityofaustin/vision-zero/pull/1874#discussion_r2411701363 
and including this check for Infinity